core/dbt/adapters/base/impl.py

```diff
@@ -1072,6 +1093,11 @@ def get_compiler(self):
 
         return Compiler(self.config)
 
+    # used in apply_grants -- True is a safe default
+    @available
+    def do_i_carry_over_grants_when_an_object_is_replaced(self) -> bool:
+        return True
+
```

Comment:
Remove since now unused. We've opted for macros instead.
```suggestion
```

Reply:
done